networking/network_observability/network-observability-overview.adoc

```diff
@@ -26,6 +26,10 @@ The Network Observability Operator requires the following Operators:
 
 The Network Observability Operator provides the Flow Collector API custom resource definition. A Flow Collector instance is created during installation and enables configuration of network flow collection. The Flow Collector instance deploys pods and services that form a monitoring pipeline where network flows are then collected and enriched with the Kubernetes metadata before storing in Loki. The eBPF agent, which is deployed as a `daemonset` object, creates the network flows. 
 
+[id="network-observability-supported-architectures"]
+=== Supported architectures
+The Network Observability Operator supports multiple architectures. It is required to have one of the following: `amd64`, `ppc64le`, or `arm64`.
```

Comment:
Same comment here

Reply:
I feel we should only keep that list in `Prerequisites`. WDYT ?

Reply:
Ok, I was wondering if it was overkill to have it in the Overview and the prerequisites. I think its fine to have it in just the prerequisites.